src/components/header/nav.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, Link } from "@mui/material";
+import { Auth } from "../../atoms/auth";
+
+interface HomeNavProps { auth: Auth | undefined}
+
+const HomeNav = (props: HomeNavProps) => {
+    const { auth } = props;
+
+    interface NavButtonProps {
```

Comment:
Types like this should be exported from a types folder- good way to reduce bloat, also needs JSDoc comment